client/components/reader-full-post/README.md

```diff
@@ -0,0 +1,3 @@
+# Reader Full Post
+
+The "new" reader full post component
```

Comment:
Worth adding a tiny bit of blurb about what the new full post view does differently?